optimizely/event/event_processor.py

```diff
@@ -59,7 +61,22 @@ def __init__(self,
                 event_queue=None,
                 batch_size=None,
                 flush_interval=None,
-                timeout_interval=None):
+                timeout_interval=None,
+                notification_center=None):
+    """ EventProcessor init method to configure event batching.
+    Args:
+      event_dispatcher: Provides a dispatch_event method which if given a URL and params sends a request to it.
+      logger: Provides a log method to log messages. By default nothing would be logged.
+      default_start: Optional boolean param which starts the consumer thread if set to True.
+                     By default thread does not start unless 'start' method is called.
+      event_queue: Optional component which accumulates the events until dispacthed.
+      batch_size: Optional param which defines the upper limit of the number of events in event_queue after which
+                  the event_queue will be flushed.
+      flush_interval: Optional param which defines the time in milliseconds after which event_queue will be flushed.
+      timeout_interval: Optional param which defines the time in milliseconds before joining the consumer
```

Comment:
I'd like for us to have any time intervals in seconds like we already have for datafile management.